compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -1,3 +1,4 @@
+// ignore-tidy-filelength
```

Comment:
:(

(not really something this PR has to fix though, but it may be nice to split this file up further in the future, maybe by moving out more diagnostic paths)